packages/paste-website/src/pages/utilities/flex/index.mdx

```diff
@@ -0,0 +1,348 @@
+    </Flex>
+    <Flex grow>
+      <Box
+        backgroundColor="colorBackgroundPrimaryLighter"
+        padding="space40"
+        width="100%"
+      >
+        Right area
+      </Box>
+    </Flex>
+  </Flex>`}
+</LivePreview>
+
+#### Wrap Property
+
+<LivePreview scope={{Box, Flex}} language="jsx">
+  {`<Flex wrap>
+    <Flex basis={800}>
+      <Box
+        backgroundColor="colorBackgroundPrimaryLight"
+        padding="space40"
+        width="100%"
+      >
+        Left area
+      </Box>
+    </Flex>
+    <Flex>
+      <Box
+        backgroundColor="colorBackgroundPrimaryLighter"
+        padding="space40"
+        width="100%"
+      >
+        Right area
+      </Box>
+    </Flex>
+  </Flex>`}
+</LivePreview>
+
+#### Vertical Alignment Properties
+
+<LivePreview scope={{Box, Flex}} language="jsx">
+  {`<Flex vAlignContent="center">
+    <Flex grow>
+      <Box
+        backgroundColor="colorBackgroundPrimaryLight"
+        padding="space40"
+        width="100%"
+      >
+        Left area
+      </Box>
+    </Flex>
+    <Flex grow>
+      <Box
+        backgroundColor="colorBackgroundPrimaryLighter"
+        padding="space40"
+        width="100%"
+        height="size10"
+      >
+        Right area
+      </Box>
+    </Flex>
+  </Flex>`}
+</LivePreview>
+
+#### Horizontal Alignment Properties
+
+<LivePreview scope={{Box, Flex}} language="jsx">
+  {`<Flex hAlignContent="center" vertical>
+    <Flex>
+      <Box
+        backgroundColor="colorBackgroundPrimaryLight"
+        padding="space40"
+        width="100%"
+      >
+        Left area
+      </Box>
+    </Flex>
+    <Flex>
+      <Box
+        backgroundColor="colorBackgroundPrimaryLighter"
+        padding="space40"
+        width="size30"
+      >
+        Right area
+      </Box>
+    </Flex>
+  </Flex>`}
+</LivePreview>
+
+### Composition Notes
+
+The Flex component is used to compose flexible layout experiences. Let's look at an example
+where we need to build a flexible content layout with two sidebars: its has a left sidebar,
+a content area, and a right sidebar. Here's how we’ll compose the flexible layout using Flex:
+
+<LivePreview scope={{Box, Flex}} language="jsx">
+  {`<Flex>
+    <Flex>
+      <Box
+        backgroundColor="colorBackgroundPrimaryLight"
+        padding="space40"
+        height="size20"
+      >
+        Left sidebar area
+      </Box>
+    </Flex>
+    <Flex grow>
+      <Box
+        backgroundColor="colorBackgroundPrimaryLighter"
+        padding="space40"
+        width="100%"
+        height="size20"
+      >
+        Content area
+      </Box>
+    </Flex>
+    <Flex>
+      <Box
+        backgroundColor="colorBackgroundPrimaryLight"
+        padding="space40"
+        height="size20"
+      >
+        Right sidebar area
+      </Box>
+    </Flex>
+  </Flex>`}
+</LivePreview>
+
+### When to use Flex
+
+The Flex component should be used when a flexible layout is needed with a page layout or custom component layout.
+
+<DoDont>
```

Comment:
I think we should always provide examples for DoDonts.  I want to change the API of that component soon to disallow `preview={false}`.

Reply:
I agree. I think we need them across the board. We have so few on the site. It'd be nice if we could find someone to do illustrations for the examples.